securityscanutils/trivy_scanner_test.go

```diff
@@ -49,7 +52,9 @@ var _ = Describe("Trivy Scanner", func() {
 	})
 
 	It("Cannot find Image", func() {
-		inputImage = "quay.io/solo-io/gloo:1.11.13245"
+		t = NewTrivyScanner(func(cmd *exec.Cmd) ([]byte, int, error) {
+			return []byte("Error containing the string 'No such image: '"), VulnerabilityFoundStatusCode + 1, eris.Errorf("Unread error")
```

Comment:
Anything that isn't `VulnerabilityFoundStatusCode` would correspond to the image not being pulled, i just went with +1 to make it look realistic but this could in theory be anything.